meshtastic/node_runtime/content_runtime.py

```
@@ -0,0 +1,515 @@
+                has_field
+            ) and has_field(  # pylint: disable=not-callable
+                "get_ringtone_response"
+            )
+        except (TypeError, ValueError):
+            has_ringtone_response = False
+        if not has_ringtone_response:
+            logger.warning("Unexpected ringtone response without raw ringtone data")
+            return (True, None)
+        try:
+            payload = raw_admin.get_ringtone_response
+            return (True, payload)
```

Ruff (0.15.7)

[warning] Consider moving this statement to an `else` block

(TRY300)